codejam-canvas/index.html

```diff
@@ -0,0 +1,53 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css">
+    <title>Codejam Canvas</title>
+</head>
+
+<body>
+    <header>
+        <div class="header__nav">
+            <nav><i class="fas fa-bars"></i></nav>
+            <h1>CodeJam-Palette</h1>
+        </div>
+        <i class="fas fa-ellipsis-v"></i>
+    </header>
+    <main>
+        <div class="tools">
+            <ul class="tools__list">
+                <li class="tools__list--item"><i class="tools fas fa-fill-drip"></i><span class="item--text">Paint
+                        bucket</span></li>
+                <li class="tools__list--item choose"><i class="tools fas fa-eye-dropper"></i></i><span
```

Comment:
use code validator or IDE extensions to prevent such  mistakes in the future `</i></i>`